pinot-spi/src/main/java/org/apache/pinot/spi/utils/TimeUtils.java

```diff
@@ -194,4 +198,20 @@ public static boolean isPeriodValid(String timeStr) {
       return false;
     }
   }
+
+  public static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+
+  public static Long convertDateTimeStringToMillis(String timeStr) {
```

Comment:
Please change this signature to use a primitive `long` instead of `Long`, since the method never returns null, change the variable declarations where this method is used to primitive `long`.